style.css

```diff
@@ -0,0 +1,542 @@
+  margin-top: .3rem;
+  margin-right: 1rem;
+  line-height: 1rem;
+}
+
+.item-cost h5 {
+  margin-bottom: 1rem;
+  border: 2px solid red;
+}
+
+.item-cost span {
+  margin-left: 1rem;
+}
+
+.total {
+  line-height: 1rem;
+
+}
+
+.submit {
+  margin-top: 1.5rem;
+  margin-bottom: 3rem;
+  padding: .5rem 5rem;
+  background-color: rgba(115, 153, 0, 1);
+  text-decoration: none;
+  color: white;
+  font-weight: bolder;
+  border: none;
+  font-size: 1rem;
+}
+
+.invisible {
+  display: block;
+  visibility: hidden;
+}
+
+footer {
+  background-color: rgba(0, 60, 100, 1);
+  color: white;
+  text-align: center;
+  padding: 3rem;
+  font-family: sans-serif;
+  font-size: .8rem;
+}
+
+#footer-logo {
+  max-width: 300px;
+}
+
+footer p {
+  line-height: 1.5rem;
+}
+
+footer nav h2 {
+    margin: 1rem;
+}
+footer nav h2 *{
+  color: white;
+  font-family: sans-serif;
+  font-size: .8rem;
+  text-decoration: none;
+  font-weight: bolder;
+  padding: .25rem;
+}
+
+footer p span{
+  margin: .5rem;
+}
+
+
+/* Media Queries */
+/* Mobile */
+@media screen and (max-width: 767px) {
+  .intro {
+    text-align: center;
+  }
+  .intro img {
+    display: inline;
+    text-align: center;
+    margin-left: auto;
+    margin-right: auto;
+  }
+  .introtext {
+    font-size: 1rem;
+    width: 90%;
+  }
+  .current-dogs-container {
+    display: block;
+    width: 100%;
+    height: auto;
+    overflow: hidden;
+  }
+  .dog-card {
+    width: 90%;
+    margin-left: 5%;
+    margin-right: 5%;
+    height: auto;
+  }
+  .dog-card img {
+    right: 2px;
+    width: 101%;
+  }
+  #blog-preview {
+    display: none;
+  }
+}
+
+/* Tablet */
+@media screen and (min-width: 768px) and (max-width: 1145px) {
+  .introtext {
+    font-size: 1rem;
+    width: 90%;
+  }
+  .current-dogs-container {
+    width: 100%;
+    height: auto;
+    overflow: hidden;
+  }
+  .dog-card {
+    width: 43%;
+    height: auto;
+  }
+  .dog-card img {
+    right: 2px;
+    width: 101%;
+  }
+
+  h5 ~ span {
+    margin: 0;
+    position: relative;
+    right: auto;
+    font-family: sans-serif;
+    font-size: .75rem;
+    color: black;
+  }
+}
+
+/* Home Page with Blog previews */
+@media screen and (min-width: 768px) {
+  .intro {
+    margin: 2rem 10%;
+    overflow: auto;
+  }
+
+  .intro img {
+    width: auto;
+    float: left;
+    margin-right: 2%;
+    margin-bottom: 4%;
+  }
+#blog-preview {
+  float: right;
+  width: 25%;
+  height: auto;
+  margin: .7rem 1% 4%;
+  overflow: hidden;
+  }
+
+#blog-preview * {
+  width: 80%;
+  height: auto;
+  margin-right: 0;
+  }
+
+#blog-preview img {
+  width: 65%;
+  height: 100px; /* height of container */
+  object-fit: cover ;
+}
+
+#preview-crop-3 {
+  object-position: 0 25%; /* reposition preview image 3 */
+}
+
+#blog-preview h3 {
+  margin: .5rem 0;
+  }
+
+#blog-preview h4 {
+  margin: .5rem 0;
+  }
+
+
+#blog-preview p {
+  line-height: 1rem;
+  }
+
+#blog-preview a h6 {
+  margin: 0;
+  }
+
+.current-dogs-container {
+  width: 60%;
```

Comment:
Would recommend changing to 80%, with 10% margin on each side it will make your content look more centered. Just a suggestion!

Reply:
Thank you for the feedback Tyler! I'll give it a go. -S

Reply:
Hello Tyler, I applied the suggested percentages, but found that a width of 80% on .current-dogs-container made the content spill to the right and encroached on the blog preview section. 63% was the most I could increase it's size comfortably. Am I interpreting your comment correctly?

Reply:
Thread resolved via email.